src/main/java/duke/backend/Storage.java

```diff
@@ -0,0 +1,53 @@
+package duke.backend;
+
+import duke.task.Task;
+import duke.task.TaskList;
+
+import java.io.*;
```

Comment:
Should imports be listed explicitly? (See Java coding standard -> Statements -> Package and Import Statements)